### DISCUSSION TITLE
https://johanneskonings.dev/notes/2020-08-03-aws-helper/

### DISCUSSION THREAD

**Discussion by giscus** [JohannesKonings/JohannesKonings.github.io#30] [Note comments]
# https://johanneskonings.dev/notes/2020-08-03-aws-helper/

AWS helper

https://johanneskonings.dev/notes/2020-08-03-aws-helper/

**Comment by JohannesKonings** [comment]
https://github.com/mingrammer/diagrams

**Comment by JohannesKonings** [comment]
https://github.com/awslabs/threat-composer

**Comment by JohannesKonings** [comment]
https://cloudcustodian.io/

**Comment by JohannesKonings** [comment]
https://www.linkedin.com/posts/theburningmonk_do-you-know-any-underrated-tools-for-building-activity-7214711026749968384-AMTp

**Comment by JohannesKonings** [comment]
https://github.com/Soname-Solutions/salmon

**Comment by JohannesKonings** [comment]
https://github.com/awslabs/security-hub-compliance-analyzer

**Comment by JohannesKonings** [comment]
https://github.com/hotnops/apeman

**Comment by JohannesKonings** [comment]
https://github.com/gabrielsoltz/my-steampipe-arsenal

**Comment by JohannesKonings** [comment]
https://github.com/aws-observability

**Comment by JohannesKonings** [comment]
https://github.com/ServerlessLife/lambda-live-debugger

**Comment by JohannesKonings** [comment]
https://jameskimthing.github.io/aws-icons

**Comment by JohannesKonings** [comment]
https://github.com/synfinatic/aws-sso-cli

**Comment by JohannesKonings** [comment]
https://awseye.com/

**Comment by JohannesKonings** [comment]
https://awsstash.com/#

**Comment by JohannesKonings** [comment]
https://github.com/udondan/cfn-teleport

**Comment by JohannesKonings** [comment]
https://github.com/fourTheorem/slic-watch

**Comment by JohannesKonings** [comment]
https://github.com/tsypuk/aws-client-monitor

**Comment by JohannesKonings** [comment]
https://github.com/dannysteenman/aws-cdk-starterkit

**Comment by JohannesKonings** [comment]
https://iam-liart.vercel.app/

**Comment by JohannesKonings** [comment]
https://github.com/FogSecurity/finders-keypers

**Comment by JohannesKonings** [comment]
https://github.com/zoph-io/kye

**Comment by JohannesKonings** [comment]
https://github.com/alexkates/dynamodb-extended

**Comment by JohannesKonings** [comment]
https://github.com/dacort/s3grep

**Comment by JohannesKonings** [comment]
https://www.fogsecurity.io/blog/aws-size-release

**Comment by JohannesKonings** [comment]
https://www.cdkbooster.com/

**Comment by JohannesKonings** [comment]
https://github.com/go-to-k/delstack

**Comment by JohannesKonings** [comment]
https://dynamodbnavigator.com/

**Comment by JohannesKonings** [comment]
https://event-driven-io.github.io/emmett/

**Comment by JohannesKonings** [comment]
https://www.npmjs.com/package/cdk-dashboard

**Comment by JohannesKonings** [comment]
https://github.com/philippemerle/AWS-CloudFormation-Diagrams

**Comment by JohannesKonings** [comment]
https://www.reddit.com/r/aws/s/3uddWbwPYJ

**Comment by JohannesKonings** [comment]
https://www.dynamolens.com/

**Comment by JohannesKonings** [comment]
https://www.dynamolens.com/

**Comment by JohannesKonings** [comment]
https://github.com/awslabs/iam-policy-autopilot

**Comment by JohannesKonings** [comment]
https://github.com/NguyenVanHung24/IAM-SCANNER

**Comment by JohannesKonings** [comment]
https://dev.to/aws-heroes/zero-orphaned-resources-force-deleting-any-cloudformation-stack-3b1h

**Comment by JohannesKonings** [comment]
https://github.com/elC0mpa/aws-doctor

**Comment by JohannesKonings** [comment]
https://github.com/fukusuket/THuntCloud
